src/Result.php

```diff
@@ -60,15 +60,37 @@ public function concat(Result $that) : Result
      */
     public function errors() : array
     {
-        return $this->errors;
+        return array_map(function (/* ValidationError */ $error) : string {
+            return (string) $error;
+        }, $this->errors);
+    }
+
+    /**
+     * Get and translate the errors for this result.
+     * @param array $translations
+     * @return array
+     */
+    public function translate(array $translations) : array
+    {
+        return array_reduce(
+            $this->errors,
+            function (array $carry, ValidationError $error) use ($translations) : array {
+                if (array_key_exists($error->token(), $translations)) {
+                    return array_merge($carry, [$error->translate($translations[$error->token()])]);
```

Comment:
I was trying to draw a fish`<))))><` but couldn't work the greater/less thans in there :(